src/init.cpp

```diff
@@ -340,8 +340,7 @@ std::string HelpMessage(HelpMessageMode mode)
 
     string strUsage = HelpMessageGroup(_("Options:"));
     strUsage += HelpMessageOpt("-?", _("This help message"));
-    strUsage += HelpMessageOpt("-addressindex", strprintf(_("bitcore: Maintain a full address index, used to query for the balance, txids and unspent outputs for addresses (default: %u)"), false));
-    strUsage += HelpMessageOpt("-spentindex", strprintf(_("Maintain a full spent index, used to query the spending txid and input index for an outpoint (default: %u)"), false));
+    strUsage += HelpMessageOpt("-addressindex", strprintf(_("bitcore: Maintain a full address index, used to query for the balance, txids and unspent outputs for addresses (default: %u)"), 0));
```

Comment:
I moved these to maintain alphabetical order (should have been done in earlier commits).